knexfile.js

```diff
@@ -1,4 +1,4 @@
-var config = require('app/config/config');
+var config = require('./app/config/config');
```

Comment:
no need for the dot, since we are setting NODE_PATH=. already [here](https://github.com/montera82/ce-berlin-non-profit/blob/master/.env#L2)

Reply:
ok cool.